inc/nu/impl/proclet.ipp

```diff
@@ -73,6 +79,26 @@ retry:
     Migrator::migrate_thread_and_ret_val<void>(
         std::move(return_buf), to_proclet_id(caller_header), nullptr, nullptr);
   }
+  else{
+    // metric logging
+    if (caller_header){
+      ProcletSlabGuard slab_guard(&caller_header->slab);
+      NodeIP target_ip = get_runtime()->rpc_client_mgr()->get_ip_by_proclet_id(id);
+
+      caller_header->spin_lock.lock();
+
+      auto target_kvpair = caller_header->remote_call_map.find(target_ip);
```

Comment:
why the key is the "target_ip" not simply the "callee_id"?

Reply:
For locality improvements we are interested in the total aggregate communication from a proclet to all other machines, using the ip of the callee allows us to collect data on a per-target-machine basis. Using callee_id as key will not be optimal since proclets can migrate